README.md

```diff
@@ -49,6 +49,11 @@ Setting up the environment
 
     ![gef installation](doc/images/eclipseInstallNewSoftwareNebula.png)
 
+-   On macOS you might need to additionally install Equinox Target Components
```

Comment:
Also needed on Windows 10, or ``org.eclipse.equinox.ds`` is not available.

Reply:
Thanks!